lambda/index.js

```diff
@@ -1,8 +1,29 @@
 const exec = require('child_process').exec;
+const fs = require('fs');
+
+exports.handler = function(event, context, callback) {
+  // In order to return a value and not deal with stdout rumbda watches a file.
+  const jsonFilePath = '/tmp/rumbda.json';
+  event['RUMBDA_RESULT_JSON_FILENAME'] = jsonFilePath;
 
-exports.handler = function(event, context) {
   const child = exec('./ruby_wrapper ' + "'" +  JSON.stringify(event) + "'", (result) => {
-    context.done(result);
+    if (fs.existsSync(jsonFilePath)) {
+      fs.readFile(jsonFilePath, 'utf8', function (err, data) {
+        if (err) {
+          fs.unlink(jsonFilePath); 
+          return callback(err);
+        }
+        if (data.length > 0) {
+          callback(null, JSON.parse(data));
+        } else {
+          callback(null, data);
+        }
+
+        fs.unlink(jsonFilePath); 
+      });
+    } else {
+      callback(null, result);
```

Comment:
I think this should be `callback(result);` since the "result" is an [error returned by exec](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback).